clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -495,6 +514,10 @@ def register_model(self,
         labels : list(str), optional
             A list of strings annotating the model. These are ignored by Clipper
             and used purely for user annotations.
+        batch_size : int, optional
+            The user defined batch size.
+            The default value is -1, which means that the user did not define a specific batch size.
```

Comment:
Update the documentation for this method in the same way as for `build_and_deploy_model`.